invenio_rdm_records/resources/resources.py

```diff
@@ -76,10 +76,89 @@ def p(route):
                 p(routes["access-request-settings"]),
                 self.update_access_settings,
             ),
+            route("POST", p(routes["delete-record"]), self.delete_record),
```

Comment:
Why `POST` and not `DELETE`?

Reply:
interesting axios quirks made me think DELETE does not accept payload, but after verifying, it is fixed, thanks!